src/dune/dune.py

```diff
@@ -657,7 +683,15 @@ def activate_feature(self, code_name, preactivate=False):
             print("Feature Not Found")
             raise dune_error()
 
-    def bootstrap_system(self, full):
+    def setup_token(self, currency, max_value, initial_value):
```

Comment:
Should we test `initial_value > max_value` and error?

Reply:
I don't think so, because nodeos handles it.
The following error is returned:
```
assertion failure with message: quantity exceeds available supply
    {"s":"quantity exceeds available supply"}
```